toolkit/themes/shared/design-system/config.js

```diff
@@ -45,7 +53,11 @@ module.exports = {
           format: "css/variables",
           options: {
             outputReferences: true,
-            showFileHeader: false,
+            fileHeader: () => [
```

Comment:
I think it'll work better for us long term to define the custom header in such a way that we can reference it in multiple formats, like how they do it in this example: https://github.com/amzn/style-dictionary/blob/main/examples/advanced/custom-file-header/build.js

If that seems out of scope here though we can just change it when we add the different formats for brand, platform, etc.